website/docs/concepts/denovo.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This file is a lot of detail. IMO, it would be better to reference the scripts themselves and have those be sufficiently organized/commented that the methods are clear.